Src/Microsoft.Scripting/Hosting/ExceptionOperations.cs

```diff
@@ -70,8 +70,7 @@ public sealed class ExceptionOperations : MarshalByRefObject {
 
         public bool HandleException(ObjectHandle exception) {
             ContractUtils.RequiresNotNull(exception, nameof(exception));
-            var exceptionObj = exception.Unwrap() as Exception;
-            ContractUtils.Requires(exceptionObj != null, "exception", "ObjectHandle must be to Exception object");
+            ContractUtils.Requires(exception.Unwrap() is Exception exceptionObj, "exception", "ObjectHandle must be to Exception object");
```

Comment:
nameof(exception) perhaps?

Reply:
also no point in assigning to `exceptionObj` if it's unused?